### ISSUE TITLE
BSIP-83: Decouple BitAssets from Platform Governance Process

### ISSUE THREAD

**Issue by pmconrad** [bitshares/bsips#239]
This is supposed to be the discussion thread for [BSIP-0083](https://github.com/bitshares/bsips/pull/238).

**Comment by bangzi1001** [comment]
As a witness I personally like this idea but it just won't work. This BSIP solve a small problem but create a bigger problem. It avoid witnesses being voted out due to price feed issue but actually terminate life of BitAssets once it pass to third party, because there is no party have higher credibility than committee and witnesses in Bitshares Blockchain. The destiny of third party smartcoins is lack of price feed providers or global settlement at the end.  

Anyway, this BSIP was not proposed by committee members, nor witnesses, nor top proxy. I don't see the possibility to get this BSIP voted in. What concern me now is next core team WP can get voted in or not.

**Comment by shulthz** [comment]
Did the authors think someting carefully and clearly?
The committee and wittness didn't get things done in bitasset, and now want to shirking its responsibility to a third party?
Why this BSIP have a number so quickly? a political game?
IS this is operating range of core team?

This BSIP seems like a joke and didn‘t clear principle of POS.

If the witnesses didn't want to feed price of bitasset really, we can design a witnesses of feed price mechanism in the blockchain.
1. The Provider of feed price need to lock a number BTS in the blockchain;
2. The Provider of feed price need to be voted in or out by the holder of bts;
3. The Provider of feed price will get the reward from the bitasset which he feed;
4. The Provider of feed price should follow the BSIP which be vote in;

**Comment by froooze** [comment]
>     1. Think of each mpa as a product that a business is selling. Competition is good.
YES

I see the need for a new group/token called price-feeder, which is independent from witnesses. 
Asset owner needs to buy price-feed-tokens and creates a whitelist for potential price-feeder.  
Price-feeder are selected and paid by asset owner to provide price feeds.

**Comment by MichelSantos** [comment]
The current text is being revised based on discussions with the community including the Committee, and will be updated in the future

**Comment by bangzi1001** [comment]
Decouple BitAssets from Bitshares BSIPS Repository 
https://github.com/bitshares/committee-tools/issues/11

**Comment by wenhuadream** [comment]
> Why this BSIP have a number so quickly? a political game?
> 
> This BSIP seems like a joke and didn‘t clear principle of POS.
> 
Such an obvious and serious problem, why didn’t  get back for such a long time?

**Comment by pmconrad** [comment]
What exactly do you mean with "obvious and serious problem"?
How can the assignment of a sequential number be a "serious problem"?
Or that someone considers this a joke?
And what do you mean with "didn't clear principle of POS"?

**Comment by wenhuadream** [comment]
> What exactly do you mean with "obvious and serious problem"?
> How can the assignment of a sequential number be a "serious problem"?
> Or that someone considers this a joke?
Why this BSIP have a number so quickly? a political game?

**Comment by pmconrad** [comment]
Because the person who was assigning BSIP numbers happened to be one of the authors.

Again, how can the simple assignment of a sequential number be a "serious problem"? And in what way could the number be used in "a political game"? IMO this is a non-issue.

**Comment by wenhuadream** [comment]
> Because the person who was assigning BSIP numbers happened to be one of the authors.
> 
> Again, how can the simple assignment of a sequential number be a "serious problem"? And in what way could the number be used in "a political game"? IMO this is a non-issue.

Obviously, this reason does not make sense. I have the power, can I do anything that is good for me? Who gives you the power? community.

I think this is a very serious act of ultra vires, taking power for personal gains.

**Comment by pmconrad** [comment]
Please explain how assigning a serial number effects "personal gains".

**Comment by wenhuadream** [comment]
> Please explain how assigning a serial number effects "personal gains".

I think that some people think that this is good for themselves, or that he wants to hurt some people in the community, which has brought a personally clear meaning. I think this is already very serious.

**Comment by pmconrad** [comment]
I understand that the ideas presented in BSIP-83 are not well-received by all community members.
However, I fail to see how the assignment of a number *in itself* can hurt or benefit anyone.

Please note that BSIP-76 did receive a number when requested, despite the fact that there was much disagreement about it: https://github.com/bitshares/bsips/issues/221#issuecomment-535153231

**Comment by wenhuadream** [comment]
> I understand that the ideas presented in BSIP-83 are not well-received by all community members.
> However, I fail to see how the assignment of a number _in itself_ can hurt or benefit anyone.
> 

Because some people want to quickly pass this BSIP in order to harm the interests of some people in the community (such as cn-vote), they are too eager to use the authority to assign numbers and want to pass this bsip.

IMHO this is entirely a power for personal gain.

**Comment by pmconrad** [comment]
The only BSIP that was passed quickly was BSIP-76.
BSIP-83 was being discussed in public for two weeks before being merged. The only things which would affect cn-vote (i. e. the voting weight change and the ownership change) was removed from the BSIP before the merge.
I don't see where the personal gain is there.

**Comment by wenhuadream** [comment]
> The only BSIP that was passed quickly was BSIP-76.
> BSIP-83 was being discussed in public for two weeks before being merged. The only things which would affect cn-vote (i. e. the voting weight change and the ownership change) was removed from the BSIP before the merge.
> I don't see where the personal gain is there.



Back to the original question

Why this BSIP have a number so quickly? a political game?
 This BSIP seems like a joke and didn‘t clear principle of POS.

**Comment by jmjatlanta** [comment]
> Why this BSIP have a number so quickly? a political game?

This was answered above, when pmconrad said: "Because the person who was assigning BSIP numbers happened to be one of the authors." Therefore, no. It was not a political game.

> This BSIP seems like a joke and didn‘t clear principle of POS.

Thank you for your opinion. However, myself and many within the community believe this BSIP is very well aligned with POS principles. In fact, it is trying to fix a broken system where the "S" of POS could be (and is being) artificially inflated by some.
